package.json

```diff
@@ -36,6 +36,8 @@
     "lint:cypress:fix": "yarn lint:cypress --fix",
     "postinstall": "git config --local include.path '.gitconfig'",
     "release": "yarn ts-node --skipProject ./scripts/release.ts",
+    "update:translation-value": "yarn ts-node --skipProject ./scripts/updateTranslationValue.ts",
+    "update:translation-path": "yarn ts-node --skipProject ./scripts/updateTranslationPath.ts",
```

Comment:
Added as a local convenience, happy to revert it if we don't want these, but I feel like these will come quite handy

Assumes `jq` is installed and is used like:

- `yarn update:translation-value defi.modals.saversVaults.protocolFee "Protocol Fee"` - updates values of `defi.modals.saversVaults.protocolFee` across all translations to `Protocol Fee`
- `yarn update:translation-path defi.modals.saversVaults.estimatedFeeTooltip defi.modals.saversVaults.protocolFeeTooltip` - updates the `defi.modals.saversVaults.estimatedFeeTooltip` keys across all translations to `defi.modals.saversVaults.protocolFeeTooltip`

Definitely a very rambo script, i.e doesn't support things like changing a path entirely, but a nice QoL companion script to update translations holistically, granted you git add interactive hunks when you run it to double check the written files

Reply:
>  across all translations to

@gomesalexandre Thanks for trying to make this easier, but I don't think this should be done in general unless we know there won't be actual translation work to do on the strings in non-English languages (the only example that comes to mind right now would be a network/chain name or a brand). In other cases I would ask again to not touch the non-English translations files.

Every time we start a "sync" with our translators the strings that have changed since the last time in English are reviewed and invalidated if needed using this existing script: https://github.com/shapeshift/web/blob/develop/scripts/removeOutdatedTranslations.ts . Once this is done we let the translators (re-)translate these strings and the new ones on Gitlocalize (our translation platform).

The changes you've made in this PR to the non-English translations files are a good example of something that isn't helping the translations process, can you please revert them? I will invalidate these old translations the next time there will be a sync with our translators.

Reply:
tyvm @firebomb1 ! Second script it is then 🙏🏽 also reused `trade.protocolFee` which is already translated - fc6ff4997016c42c144f07a2a870f78382f6cf58